plugin/src/test/java/org/wso2/carbon/uuf/maven/bean/AppConfigTest.java

```diff
@@ -105,18 +105,18 @@ public void testMenuValidations() {
 
     @Test
     public void testSecurityConfigValidations() {
-        AppConfig.PatternsConfig patternsConfig = new AppConfig.PatternsConfig();
+        AppConfig.SecurityConfig securityConfig = new AppConfig.SecurityConfig();
         Assert.assertThrows(IllegalArgumentException.class,
-                            () -> patternsConfig.setAccept(Collections.singletonList("")));
+                () -> securityConfig.setCsrfIgnoreUris(Collections.singletonList("")));
         Assert.assertThrows(IllegalArgumentException.class,
-                            () -> patternsConfig.setReject(Collections.singletonList("")));
-
-        patternsConfig.setAccept(null);
-        patternsConfig.setAccept(Collections.emptyList());
-        patternsConfig.setAccept(Collections.singletonList("/some/pattern"));
-        patternsConfig.setReject(null);
-        patternsConfig.setReject(Collections.emptyList());
-        patternsConfig.setReject(Collections.singletonList("/some/pattern"));
+                () -> securityConfig.setCsrfIgnoreUris(Collections.singletonList("some/uri")));
+        Assert.assertThrows(IllegalArgumentException.class,
+                () -> securityConfig.setXssIgnoreUris(Collections.singletonList("")));
+        Assert.assertThrows(IllegalArgumentException.class,
+                () -> securityConfig.setXssIgnoreUris(Collections.singletonList("some/uri")));
+
+        securityConfig.setCsrfIgnoreUris(Collections.singletonList("/valid/uri"));
```

Comment:
"CSRF ignore URI"s and "XSS ignore URI"s can be `null`, and its acceptable. We need to test it here.